app/Http/Controllers/StudentsCouncil/AnonymousQuestionController.php

```diff
@@ -90,14 +86,15 @@ public function store(Request $request, Semester $semester)
         }
         $validator->validate();
 
-        $event = $this->periodicEventForSemester($semester);
+        $event = $this->periodicEvent();
 
-        $question = $semester->questions()->create([
+        $question = $event->semester->questions()->create([
             'title' => $request->title,
             'max_options' => $hasLongAnswers ? 0 : $request->max_options,
             'has_long_answers' => $hasLongAnswers,
-            'opened_at' => $event?->start_date ?? null,
-            'closed_at' => $event?->end_date ?? null
+            // does not have individual dates, voting is constrained by the periodicEvent
+            'opened_at' => null,
+            'closed_at' => null
```

Comment:
I'm just wondering whether we could use these dates for something useful. But I don't have any ideas; resolve this if neither do you.